Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.Action.ps1

```diff
@@ -36,10 +47,12 @@ try {
     Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "ProjectsJson=$projectsJson"
     Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "ProjectDependenciesJson=$projectDependenciesJson"
     Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "BuildOrderJson=$buildOrderJson"
+    Add-Content -Encoding UTF8 -Path $env:GITHUB_OUTPUT -Value "IsPartialBuild=$isPartialBuild"
 
```

Comment:
I think this is a confusing name - is should be something like buildAllProjects IMO (reversed meaning) - partial builds could be within projects and might be something we would do in the future.

Reply:
Renamed to `BuildAllProjects`.